server/plugin/command.go

```diff
@@ -313,10 +324,30 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		if err := p.SubscribeOrg(ctx, githubClient, args.UserId, owner, args.ChannelId, features, flags); err != nil {
 			return err.Error()
 		}
-
-		return fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		var subOrgMsg = fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		if isExcludeRepo {
+			var excludeMsg string
+			for _, val := range strings.Split(excludeRepo, ",") {
+				notificationOffRepoOwner, NotificationOffRepo := parseOwnerAndRepo(val, p.getBaseURL())
+				if notificationOffRepoOwner != owner {
+					return fmt.Sprintf("--exclude repository  %s is not of subscribed organization .", NotificationOffRepo)
+				}
```

Comment:
Should the exclude flag accept a list of repos instead of a list of org/repo?

Reply:
currently it is taking org/repo ,
do we want to take that feature ?

Reply:
The feature seems crucial to me as subscriptions usually happen on an org level for this plugin.

Reply:
subscriptions can be done on both org and individual repos , but excluding repos can only be done it we are doing in org level.

so do we want to add this feature in repo level too ?